app/jobs/bulk_invoice_job.rb

```diff
@@ -8,13 +8,9 @@ class BulkInvoiceJob < ApplicationJob
   def perform(order_ids, filepath, options = {})
     @options = options
 
-    orders = Spree::Order.where(id: order_ids)
+    # The `find` method returns records in the same order as the given ids.
+    orders = Spree::Order.find(order_ids)
 
-    if OpenFoodNetwork::FeatureToggle.enabled?(:invoices, current_user)
-      orders = orders.invoiceable
-    end
```

Comment:
The OrdersReflex triggering this job was already applying this filter regardless of the feature toggle.